local-dev/skaffold.yaml.template

```diff
@@ -7,6 +7,16 @@ build:
     context: ../
     jib: {}
 deploy:
-  kustomize:
-    paths:
-    - terra-config/NAMESPACE/terra-workspace-manager
+  helm:
+    releases:
+      - name: workspacemanager-ENV
+        namespace: terra-ENV
```

Comment:
I might be missing something, but aren't our namespaces currently just ENV instead of "terra-ENV"?

Reply:
Yep, this is a change we are consciously making. Also it allows a smooth transition between the kustomize-managed and helm-managed environments. @choover-broad proposed adding the terra- prefix to easily identify namespaces that correspond to Terra environments vs other k8s deployments and I agree.